### ISSUE TITLE
Language indicates focus on General election in 2018, but want to use in Primary, too

### ISSUE THREAD

**Issue by MainStreamDanny** [gpsimpact/relational_gotv#56]
I see the text in various places, which may be a placeholder, refers to the General election in 2018. Voting by mail, voting, etc all references that date.

Is this a variable, and is the system set up to run for the primary _and_ the general election, or just one? Ideally, an ambassador would be able to add and follow a contact all the way through both, and also add a contact in August for the general only (if they get greedy for points, or we add a new partner, or a partner adds a new ambassador, or a voter steps up to be an ambassador).

We definitely want to use it for the primary in August.

**Comment by jakelowen** [comment]
So currently the tool is set up for only 1 election. However there is nothing magical about the visible language which denotes general vs primary.  I might propose that we just hard code everything right now for the primary. After the primary it would just be 20 mins worth of work to update language to reflect general and we can deploy. These simple language changes won't affect any of the records in the database. 

If this is agreeable, please go ahead and close this issue and open other issues with specific language changes to update for primary.

If we are to do this however it does raise a couple minor issues with the database, which I will address in separate issues.

**Comment by MainStreamDanny** [comment]
Our concept for the project was to run it through the entire election season. So, tracking a voter from registration through two voting events (primary and general). After all, the most engaged voters are those that vote in primaries, too. Do you see any way to persist the data and engagement through both? Registration is once only. But VBM and Voted would clearly need to be duplicated and differentiated (vbm_primary2018 and vbm_general2018, for example). They could be swapped in once the primary is over... but that would probably have to be hard-coded for now, as you said.

In other words, behind the scenes (in the data) we'd want to know that the voter had voted in the primary, even though we had moved on to the general. The ambassador should still have the points for the voter voting in the primary.

But for the UI, we could only surface one VBM and one Voted? at a time, depending n if we are in the primary or general election season.

Did I just re-state what you said? :)

**Comment by jakelowen** [comment]
Yes.. We are one the same page. I have now refactored the database to have columns to store both general and primary activity. I've also reoriented the UI to display primary info. After the primary we can quickly update the ui to be hardcoded for the general. With the new DB columns already in place it will be a quick refactor and we wont have dataloss for either cycle.  Closing this for now.
